bundles/org.openhab.binding.teslapowerwall/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -69,101 +73,169 @@
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Energy</tag>
+		</tags>
 		<state readOnly="true" pattern="%.1f %unit%"/>
 	</channel-type>
 	<channel-type id="battery-energy-exported">
 		<item-type>Number:Energy</item-type>
 		<label>Battery Energy Exported</label>
 		<description>Total battery energy exported</description>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Energy</tag>
+		</tags>
 		<state readOnly="true" pattern="%.1f %unit%"/>
 	</channel-type>
 	<channel-type id="home-energy-exported">
 		<item-type>Number:Energy</item-type>
 		<label>Home Energy Exported</label>
 		<description>Total home energy exported</description>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Energy</tag>
+		</tags>
 		<state readOnly="true" pattern="%.1f %unit%"/>
 	</channel-type>
 	<channel-type id="solar-energy-exported">
 		<item-type>Number:Energy</item-type>
 		<label>Solar Energy Exported</label>
 		<description>Total solar energy exported</description>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Energy</tag>
+		</tags>
 		<state readOnly="true" pattern="%.1f %unit%"/>
 	</channel-type>
 	<channel-type id="grid-energy-imported">
 		<item-type>Number:Energy</item-type>
 		<label>Grid Energy Imported</label>
 		<description>Total grid energy imported</description>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Energy</tag>
+		</tags>
 		<state readOnly="true" pattern="%.1f %unit%"/>
 	</channel-type>
 	<channel-type id="battery-energy-imported">
 		<item-type>Number:Energy</item-type>
 		<label>Battery Energy Imported</label>
 		<description>Total battery energy imported</description>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Energy</tag>
+		</tags>
 		<state readOnly="true" pattern="%.1f %unit%"/>
 	</channel-type>
 	<channel-type id="home-energy-imported">
 		<item-type>Number:Energy</item-type>
 		<label>Home Energy Imported</label>
 		<description>Total home energy emported</description>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Energy</tag>
+		</tags>
 		<state readOnly="true" pattern="%.1f %unit%"/>
 	</channel-type>
 	<channel-type id="solar-energy-imported">
 		<item-type>Number:Energy</item-type>
 		<label>Solar Energy Imported</label>
 		<description>Total solar energy imported</description>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Energy</tag>
+		</tags>
 		<state readOnly="true" pattern="%.1f %unit%"/>
 	</channel-type>
 	<channel-type id="full-pack-energy">
 		<item-type>Number:Energy</item-type>
 		<label>Battery Full Pack Energy</label>
 		<description>Battery full pack energy</description>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Energy</tag>
+		</tags>
 		<state readOnly="true" pattern="%.1f %unit%"/>
 	</channel-type>
 	<channel-type id="degradation">
 		<item-type unitHint="%">Number:Dimensionless</item-type>
 		<label>Battery Degradation</label>
 		<description>Current battery degradation %, based on 13.5kW full capacity</description>
+		<tags>
+			<tag>Control</tag>
```

Comment:
I don't think control is right here - suggest Measurement as it's a measurement of battery degradation